bigbluebutton-html5/imports/ui/components/note/service.js

```diff
@@ -60,15 +42,39 @@ const getNoteParams = () => {
     }
   }
   return params.join('&');
-}
+};
+
+const isLocked = () => {
+  const meeting = Meetings.findOne({});
```

Comment:
```suggestion
  const meeting = Meetings.findOne({ meetingId: Auth.meetingID });
```

Otherwise you'd have issues with Breakouts imo